etc/telegraf.conf

```diff
@@ -860,6 +860,18 @@
 #   ## Master node.
 #   cluster_stats = false
 #
+#   ## Set indices_stats to true when you want to obtain indices stats from the Master node.
+#   indices_stats = false
+#
+#   ## Set shards_stats to true when you want to obtain shards stats from the Master node.
+#   ## If set, then indices_stats is considered true as they are also provided with shard stats.
+#   shards_stats = false
+
+#   ## Multiplier of the elasticsearch interval to be used to reduce the frequency of
+#   ## indices_stats and shards_stats reports. The default interval is 10 seconds, and this
+#   ## multiplier is defaulted to 6 to cause these metrics to be taken once per minute by default.
+#   indices_interval_multiplier = 6
```

Comment:
@sparrc Where are we at with this PR? I'm unsure of what changes I may need or whether you'd be accepting the change due to the other elasticsearch change thats open now.